lib/analyses/converter.rb

```diff
@@ -5,6 +5,10 @@
 
 module Analyses
   class Converter
+    def self.logger
```

Comment:
Can you add some kind of spec for this file? It only consists of public class methods and those should be tested.

If time permits, I recommend refactoring this file into instance based code, so only the intended interface is public and all other methods are private. This way a reader can instantly see which methods are meant to be called from the outside.

Another good thing would be reworking the variable and method naming. Especially in the larger codeblocks you added, you use a lot of very short abbreviations that don't make sense to anyone but you. This harms readability a lot.